luma/lcd/const.py

```diff
@@ -6,3 +6,8 @@
 class pcd8544(object):
     DISPLAYON = 0x0C
     DISPLAYOFF = 0x08
+
+
+class st7735(object):
```

Comment:
Not sure whether this const's idea is such a good idea, as there are ssd1306 specifics in core. Am in the process of detangling it, so this class is likely to be deleted at some point